portfolio/src/main/java/com/google/sps/servlets/CreatePuzzleServlet.java

```diff
@@ -51,15 +51,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     // Get the difficulty indicated by user.
     Puzzle.Difficulty difficulty;
     try {
-        difficulty = Puzzle.Difficulty.valueOf(request.getParameter("difficulty"));
+        String diff = request.getParameter("difficulty").toUpperCase();
```

Comment:
Ideally we shouldn't have to modify the request parameter in any way. Would it be possible to have the UI send parameters that match the enum?